### PR TITLE
Update Academia.edu.xml

### PR DESCRIPTION
**Pull request by bari2**

### CODE REVIEW

**Comment by Bisaloo** [comment]
Thank you for your pull request!

We try to avoid as much as possible wildcards in the rulesets. We prefer listing all working subdomains/targets. It makes review and maintaining rulesets easier.

You will find more about this and a lot of valuable information in the [HTTPS-Everywhere contributing guidelines](https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#left-wildcards).

**Comment by bari2** [comment]
@Bisaloo There are [thousands](https://ipv4.google.com/search?q=site:*.academia.edu&gbv=1&prmd=ivns&ei=uMzOWJTTNYr5UKrmmcAJ&start=10&sa=N) of subdomains for academia.edu, all those that I tested loaded with HTTPS. Here are some examples if you want to test:

    ucl.academia.edu/
    metu.academia.edu/ 
    eap.academia.edu/
    sydney.academia.edu/
    uct.academia.edu/
    ankara.academia.edu/
    bilkent.academia.edu/
    unige.academia.edu/ 
    reading.academia.edu
    univdhaka.academia.edu/

**Comment by Bisaloo** [comment]
You are right and I should have checked. In this case, the best option is indeed to use a wildcard.

**Comment by jeremyn** [comment]
This is a high profile site and the changes in the ruleset greatly expand coverage. I am uncomfortable using a wildcard here. For example, https://support.academia.edu has an invalid certificate.

I sent a message to Academia.edu through http://support.academia.edu asking them to take a look at this pull request and help us out. Hopefully they can tell us which subdomains work with HTTPS and which do not.

**Comment by bari2** [comment]
Thanks @jeremyn 

I added an exception for "http://support.academia.edu"

File: src/chrome/content/rules/Academia.edu.xml [hunk 1]
```diff
@@ -22,11 +22,15 @@
...
-	<rule from="^http://(independent\.|www\.)?academia\.edu/"
-		to="https://$1academia.edu/" />
+	
+	<test url="http://princeton.academia.edu/" />
+	<test url="http://columbia.academia.edu/" />
+	<test url="http://oxford.academia.edu/" />
+	<test url="http://reading.academia.edu/" />
+	<test url="http://support.academia.edu/" />
+	
+	<exclusion pattern="^http://support\.academia\.edu/"/>
```
Comment (J0WI) [inline src/chrome/content/rules/Academia.edu.xml]: blog.academia.edu
mail.academia.edu

Comment (bari2) [inline src/chrome/content/rules/Academia.edu.xml]: @J0WI blog.academia.edu supports HTTPS, https://blog.academia.edu/

mail.ac... doesn't so I'll add an exclusion for it.

**Pull request merged**